submissions/onyinyeChukwuka/css/sign-up.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign-up</title>
+    <link href="sign-up.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
+</head>
+<body>
+        <div class="header">
+                <nav>
+                    <div class="left-side">
+                        <div class = "tab">
+                            <strong>BOOTSTRAPS</strong>
+                        </div>
+                    </div>
+                    <div class="right-side">
+                        <div class="link-wrapper">
+                            <a href= "../html/index.html"> Product Page </a>
+                        </div>
+                        <div class="link-wrapper">
+                            <a href= "../html-forms/contactform.html">Contact Us</a>
+                        </div> 
+                        <div class="link-wrapper">
+                            <a href= "../css/sign-up.html">Sign-up</a>
+                        </div>
+                    </div>
+                </nav>
+        </div>
+        <main>
+            <div class="presentation">
+                <div class="sidebar">
+                <p><h4> Register your<br> Account</h4></p>
+                <p> Registering an account gives you<br> access to sweet promo and events<br> available to only registered<br> customers </p>
+                <p> Already got an account? <strong>Sign in</strong> </p> 
+                </div>
+                <div class="form">
+                    <form>
+                        <label for="firstname">First Name</label><br>
+                        <input type="text" class="texts" id="firstname" name="firstname"><br><br>
```

Comment:
@Onyinye101 Remove all `<br>` used in this file. Use css to style the input fields.